### DISCUSSION TITLE
#introduction

### DISCUSSION THREAD

**Discussion by Vijaylachamsetty** [community/community#67233] [New to GitHub]
Hi this is Vijay, I am very excited to start my journey as  GitHub Global Campus student hoping this jouney turned to be great one.

**Comment by melvincwng** [comment]
Welcome to GitHub Community @Vijaylachamsetty 👋 

FYI - you can also introduce yourself in the [Main Introduction Thread](https://github.com/orgs/community/discussions/23204).
